packages/nodejs/src/config/configmap.ts

```diff
@@ -56,6 +56,7 @@ export const PRIVATE_ENV_MAPPING: { [key: string]: string } = {
   _APPSIGNAL_PUSH_API_KEY: "pushApiKey",
   _APPSIGNAL_RUNNING_IN_CONTAINER: "runningInContainer",
   _APPSIGNAL_SEND_ENVIRONMENT_METADATA: "sendEnvironmentMetadata",
+  _APPSIGNAL_SEND_PARAMS: "sendParams",
```

Comment:
Does the agent need to know about this? If not, we don't need to define the environment variable (and we don't need to test for its presence, either)

Reply:
Not at this moment. I'll remove the private env var assignation from this PR, but as we talked, we might consider moving this feature to the agent.